docs/guides/styles-and-resources/resources.md

```diff
@@ -290,7 +290,7 @@ Avalonia provides different options to access Resources from code.
 ```cs
 // In this sample we have defined the resource in App.axaml and we want to look up the value in the MainWindow constructor.
 //
-//    </Application.Resources>
+//    <Application.Resources>
```

Comment:
This here was the main reason I started this edit, and then I noticed the other inconsistencies